protos/sigstore_verification.proto

```diff
@@ -162,3 +162,164 @@ message Input {
         // provided.
         optional Artifact artifact = 4;
 }
+
+// CertificateSummary contains verified metadata from the certificate
+message CertificateSummary {
+        // The issuer of the certificate
+        string certificate_issuer = 1;
+
+        // The Subject Alternative Name, or SAN, from the certificate
+        string subject_alternative_name = 2;
+
+        // The OIDC issuer. Should match `iss` claim of ID token or, in the case of
+        // a federated login like Dex it should match the issuer URL of the
+        // upstream issuer. The issuer is not set the extensions are invalid and
+        // will fail to render.
+        // OID 1.3.6.1.4.1.57264.1.8 and 1.3.6.1.4.1.57264.1.1 (Deprecated)
+        string issuer = 3;
+
+        // Deprecated
```

Comment:
The deprecated OIDs, can we ignore them here? They were marked as deprecated two years go (almost to the date) https://github.com/sigstore/fulcio/pull/945.

If a specific client want's to support them, that's fine. But from a standard perspective, I think we should only focus on the current extensions.